src/evict.c

```diff
@@ -708,3 +708,16 @@ int performEvictions(void) {
     return result;
 }
 
+int performEvictions(void) {
+    int result = doPerformEvictions();
+
+    if (result == EVICT_RUNNING) {
+        if (server.stat_last_eviction_exceeded_time == 0) {
+            server.stat_last_eviction_exceeded_time = ustime();
+        }
+    } else if (server.stat_last_eviction_exceeded_time != 0) {
+        server.stat_eviction_exceeded_time += (ustime() - server.stat_last_eviction_exceeded_time);
```

Comment:
i think the use of `ustime()` is actually wrong. we now have a monotonic clock, and that's what we should use to track (and sum) execution time (same as we now do in `call` for commandstats.

maybe we can also use the monotonic clock for current_eviction_exceeded_time?